### ISSUE TITLE
Internet sharing is not working. 

### ISSUE THREAD

**Issue by yonggeun** [SelfControlApp/selfcontrol#306]
I am using my macbook air as internet hotspot coz my company banned any personal hotspot in office. And I realize that internet sharing is not working well after I activated the app. 

I still can get an mail update throughout my iphone using internet sharing from macbook air. but safari on iphone is not working at all. 

Anyone with same problem? My setting are as below. 

![02](https://cloud.githubusercontent.com/assets/7887740/11290112/8b453f06-8f6e-11e5-818e-670fccd9e4af.jpg)
![01](https://cloud.githubusercontent.com/assets/7887740/11290113/8b4a2dc2-8f6e-11e5-933f-214c3dbd866d.jpg)

**Comment by tkosaka1976** [comment]
Me too.
It looks like Self control blocks access via sharing way.
We need a solution.
Please consider this.

**Comment by ValentinKuznetsov** [comment]
Confirming this, it blocks sharing.

Moreover, after the timer expires, sharing still does not work and nothing helps; excluding the OS complete reboot.

**Comment by jacobmilli10** [comment]
@ValentinKuznetsov yes it blocks sharing only way to stop is to reset sharing. no need to reset OS.

**Comment by hivestrung** [comment]
> @ValentinKuznetsov yes it blocks sharing only way to stop is to reset sharing. no need to reset OS.

This works for me. Also note, the internet that's shared is able to access hosts that were blocked (e.g. if you blocked facebook on your MacBook and then shared internet with your phone, your phone will be able to access facebook).
